cookbooks/fail2ban/attributes/default.rb

```diff
@@ -0,0 +1,279 @@
+# fail2ban.conf
+#          1 = ERROR
+#          2 = WARN
+#          3 = INFO
+#          4 = DEBUG
+default['fail2ban']['loglevel'] = 'DEBUG'
+default['fail2ban']['socket'] = '/var/run/fail2ban/fail2ban.sock'
+default['fail2ban']['logtarget'] = '/var/log/fail2ban.log'
+default['fail2ban']['pidfile'] = '/var/run/fail2ban/fail2ban.pid'
+
+# global config
+# XXX /!\ fail2ben 0.8.13 are comming and after 0.9.0 ... and this change some stuff in the jails
+# Available versions for v5 are:
+# - 0.8.14
+# - 0.9.3-r1
+# - 0.9.4
+default['fail2ban']['version'] = '0.9.4'
+
+# jail.local
+default['fail2ban']['jails'] = {
+  # defined using space separator.
+  'ignoreip'    => '127.0.0.1/8',
+  # "bantime" is the number of seconds that a host is banned.
+  'bantime'     => 600,
+  # A host is banned if it has generated "maxretry" during the last "findtime"
+  # seconds.
+  'findtime'    => 600,
+  # "maxretry" is the number of failures before a host get banned.
+  'maxretry'    => 3,
+  'pidfile'     => '/var/run/fail2ban/fail2ban.pid',
+  'ignorecommand' => '',
+  # "usedns" specifies if jails should trust hostnames in logs,
+  #   warn when DNS lookups are performed, or ignore all hostnames in logs
+  #
+  # yes:   if a hostname is encountered, a DNS lookup will be performed.
+  # warn:  if a hostname is encountered, a DNS lookup will be performed,
+  #        but it will be logged as a warning.
+  # no:    if a hostname is encountered, will not be used for banning,
+  #        but it will be logged as info.
+  'usedns'      => 'warn',
+  # gamin:   requires Gamin (a file alteration monitor) to be installed. If Gamin
+  #          is not installed, Fail2ban will use polling.
+  # polling: uses a polling algorithm which does not require external libraries.
+  # auto:    will choose Gamin if available and polling otherwise.
+  'backend'     => 'auto',
+  # mail configuration
+  'mail'      => {
+    'destination'   => 'security_control@company.com',
+    'sender'        => 'root@localhost'
+  },
```

Comment:
for email alerting to work the 'ssmtp' recipe has to be enabled.  Worth adding a note on the Readme.